core/src/main/java/org/apache/spark/memory/SparkOutOfMemoryError.java

```diff
@@ -39,11 +39,17 @@ public SparkOutOfMemoryError(OutOfMemoryError e) {
     }
 
     public SparkOutOfMemoryError(String errorClass, String[] messageParameters) {
-        super(SparkThrowableHelper.getMessage(errorClass, messageParameters, ""));
+        super(SparkThrowableHelper.getMessage(errorClass, null,
```

Comment:
```suggestion
        super(SparkThrowableHelper.getMessage(errorClass, None,
```

Reply:
@cloud-fan This is Java, does it have None?